acme/actors/acmestoragemigrator/actor.py

```diff
@@ -0,0 +1,43 @@
+from leapp.actors import Actor
+from leapp.models import Report, AcmeStorageInfo
+from leapp.tags import ApplicationsPhaseTag, IPUWorkflowTag
+
+from leapp import reporting
+
+
+import os
+
+
+class AcmeStorageMigrator(Actor):
+    """
+    Migrate ACME Storage device from old location to the new one
+    """
+
+    name = 'acme_storage_migrator'
+    consumes = (AcmeStorageInfo,)
+    produces = (Report,)
+    tags = (ApplicationsPhaseTag, IPUWorkflowTag)
```

Comment:
```suggestion
    tags = (ThirdPartyApplicationsPhaseTag, IPUWorkflowTag)
```

Note: this one is expected to be used for non-RH stuff. Do not forget to change import.